i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/go-web.mdx

```diff
@@ -115,9 +88,21 @@ func (storage *SessionStorage) SetItem(key, value string) {
 }
 ```
 
-## 准备前置中间件
+其中 Logto SDK 的 `Storage` 接口定义如下：
+
+```go
+// github.com/logto-io/client/storage.go
+pacakge client
+
+type Storage interface {
+	GetItem(key string) string
+	SetItem(key, value string)
+}
+```
```

Comment:
nit: Move it to the top?  As the above implementation is following this interface definition